### ISSUE TITLE
mdb-export truncates memo field data > 16k

### ISSUE THREAD

**Issue by rob-hills** [mdbtools/mdbtools#118]
Hi,
I am using mdb-export to convert data from a WebWiz forum stored in a MSAccess DB (Jet4).

The table containing post text data stores this in a memo field.  I have found that where the data stored in this field exceeds 16384 characters its output is truncated by mdb-export.

Is this a bug or a known limitation?  If it is a known limitation, should it be documented somewhere?

I can post an example DB if required.

**Comment by 00cpxxx** [comment]
Can you please post the example database?

**Comment by rob-hills** [comment]
Happy to provide the DB to the maintainer or a committer.  We have developed a fix for this problem so when I get some time, I'll put together a pull request.

**Comment by duhaime** [comment]
@rob-hills Do you still have access to your fix for this problem? If so, it'd be great if you could send a pull request!

**Comment by rob-hills** [comment]
Hi @duhaime .  Yes, I still have access to the fix.  I will bone up on pull requests and try and get one happening.

**Comment by duhaime** [comment]
@rob-hills Great news! Yes, you should fork the repository to your github user's account. Then clone your fork to your local machine, then make your changes in that newly cloned copy of the code. Next, do a `git checkout -b MY_BRANCH_NAME` where `MY_BRANCH_NAME` is the name of the feature you're working on (words should be hyphen separated). Finally, you should be able to `git push origin MY_BRANCH_NAME`. Then if you go to your forked repository, you should see your branch listed under the repository's branches. Click on the branch and click the big green button that says "Create Pull Request" then you'll be able to create a pull request that can be merged into mdbtools' copy of the repo. If you get stuck just ping me on here and I can try to assist...

**Comment by rob-hills** [comment]
Hi @duhaime, I have now created a pull request.  As the changes were very small and simple, I forked the repository to my account as you suggested, but then:

- created my branch in my forked repository via the GitHub web interface,  
- made the 3 small changes by directly editing the source in the web interface; and then
- created the change request as you specified.

Hope this all works OK for you.
